0.59.md

```diff
@@ -0,0 +1,348 @@
+Over the coming months, there will be many more components following this path to a leaner core. We're looking for help with this &mdash; head over to the [lean core umbrella](https://github.com/facebook/react-native/issues/23313) to pitch in.
+
+## CLI improvements
+
+React Native's command like tools are developer's entrypoint to the ecosystem, but they had long-standing issues and lacked official support. The CLI tools have been moved to a separate repository, and a dedicated group of maintainers have already made some exciting improvements.
+
+Logs are formatted much better now. Commands now run nearly instantly &mdash; you'll immediately notice a difference:
+
+![0.58's CLI is slow to start](resources/0.58-cli-speed.png)![0.58's CLI is nearly instananeous](resources/0.59-cli-speed.png)
+
+Some long-standing issues have been fixed too. Thanks to the [dedicated group of maintainers](https://blog.callstack.io/the-react-native-cli-has-a-new-home-79b63838f0e6) working in a [new repository](https://github.com/react-native-community/react-native-cli). This work was proposed in [RFC0003](https://github.com/react-native-community/discussions-and-proposals/blob/master/proposals/0003-Extracted-Command-Line-Tools.md), and we're excited to see it completed.
+
+## Changelog
+
+Without further adieu, the changes are below.
+
+### Added
+
+- Add Network Security Config file (fixes #22375) (#23105) ([3b0b7ce](https://github.com/facebook/react-native/commit/3b0b7ce) by [@Salakar](https://github.com/Salakar))
+- Add image bytes property of UIImage (#22731) ([103880b](https://github.com/facebook/react-native/commit/103880b) by [@zhongwuzw](https://github.com/zhongwuzw))
+- Add ImageIO related C nullable check to prevent crash (#23186) ([d0cd3ca](https://github.com/facebook/react-native/commit/d0cd3ca) by [@zhongwuzw](https://github.com/zhongwuzw))
+- Add rejectResponderTermination prop to TextInput (#16755) ([11df0ea](https://github.com/facebook/react-native/commit/11df0ea) by [@cmcewen](https://github.com/cmcewen))
+- Add QPL marker to track time it takes to load .so file from RN ([10b5218](https://github.com/facebook/react-native/commit/10b5218) by [@mdvacca](https://github.com/mdvacca))
+- Add `YGMarkerLayout` ([1db5580](https://github.com/facebook/react-native/commit/1db5580) by [@davidaurelio](https://github.com/davidaurelio))
+- Add function to set marker callbacks ([43601f1](https://github.com/facebook/react-native/commit/43601f1) by [@davidaurelio](https://github.com/davidaurelio))
+- Add `MarkerSection` ([f1a3137](https://github.com/facebook/react-native/commit/f1a3137) by [@davidaurelio](https://github.com/davidaurelio))
+- Adds fixes for react-native-windows UWP (#848) ([dfcbf97](https://github.com/facebook/react-native/commit/dfcbf97) by [@rozele](https://github.com/rozele))
+- Add QPL marker to track time it takes to load .so file from RN ([8110909](https://github.com/facebook/react-native/commit/8110909) by [@mdvacca](https://github.com/mdvacca))
```

Comment:
There's the same description above with a different commit.
Maybe we can merge them into a single line?